src/Nethermind/Nethermind.Db/IPruningConfig.cs

```diff
@@ -37,7 +37,7 @@ public interface IPruningConfig : IConfig
         FullPruningTrigger FullPruningTrigger { get; set; }
 
         [ConfigItem(
-            Description = "'Full' pruning: Defines how many parallel tasks and potentially used threads can be created by full pruning. 0 - number of logical processors, 1 - full pruning will run on single thread. " +
```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.paralleloptions.maxdegreeofparallelism?view=net-7.0 -1 in .NET means Environment.ProcessorCount